src/host/ledger.h

```diff
@@ -147,12 +148,12 @@ namespace asynchost
     }
 
     // Used when recovering an existing ledger file
-    LedgerFile(const std::string& dir, const std::string& file_name) :
+    LedgerFile(const std::string& dir, const std::string& file_name_) :
       dir(dir),
-      file(nullptr)
+      file_name(file_name_)
     {
-      auto full_path = (fs::path(dir) / fs::path(file_name));
-      file = fopen(full_path.c_str(), "r+b");
+      auto file_path = (fs::path(dir) / fs::path(file_name));
+      file = fopen(file_path.c_str(), "r+b");
```

Comment:
There's a few more s/full_path/file_path/ but the github editor won't let me change them from here.

Reply:
Thanks! It should be fixed now 👍